diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter2/comments/CommentsRule.kt

```diff
@@ -135,6 +135,9 @@ class CommentsRule(configRules: List<RulesConfig>) : DiktatRule(
         return codeFileStartCases.any { textWithoutCommentStartToken.contains(it) }
     }
 
+    private fun isRequirePartOfCode(text: String): Boolean =
+            text.contains("val", true) || text.contains("var", true) || text.contains("=", true) || (text.contains("{", true) && text.substringAfter("{").contains("}", true))
```

Comment:
```suggestion
            text.contains("val ", true) || text.contains("var ", true) || text.contains("=", true) || (text.contains("{", true) && text.substringAfter("{").contains("}", true))
```
we had spaces in the regex too

Reply:
Fixed